src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -142,11 +148,15 @@ export function getTypes(indices) {
 
 export function getIndices(includeAliases) {
   const ret = [];
-  $.each(perIndexTypes, function (index) {
-    ret.push(index);
+  Object.keys(perIndexTypes).forEach((index) => {
+    // ignore .ds* indices in the suggested indices list.
+    if (!index.startsWith('.ds')) {
```

Comment:
@zhongnansu what's stopping a customer from using an index that starts with `.ds`? Have we protected that pattern in any way?

Reply:
This is a mistake, it shouldn't be here, removing